CMakeLists.txt

```diff
@@ -338,6 +338,13 @@ if(UNITTEST)
              set_tests_properties(test_horus_binary PROPERTIES
              PASS_REGULAR_EXPRESSION "1C9A9545")
 
+     # same, but with audio as stereo
+     add_test(NAME test_horus_binary_mode_q
+             COMMAND sh -c "sox ${CMAKE_CURRENT_SOURCE_DIR}/wav/horus_binary_ebno_4.5db.wav -c 2 -r 48000 -t raw - | $<TARGET_FILE:horus_demod> -q -m binary - -"
+             )
+             set_tests_properties(test_horus_binary_mode_q PROPERTIES
+             PASS_REGULAR_EXPRESSION "1C9A9545")
+
```

Comment:
This tests that the code runs but simply copying the I to the Q doesn't test the ability of this option to demodulate complex valued signals, it's a real valued signal input.